### PR TITLE
Add note that the service is charged for

### PR DESCRIPTION
**Pull request by paulb-elastic**
Update the docs to clarify the service is charged for

### CODE REVIEW

**Comment by paulb-elastic** [comment]
@drewpost take a look at the copy, it's set as a standard block in use on 3 pages, and you can see an example of them from the [preview link](https://observability-docs_3262.docs-preview.app.elstc.co/guide/en/observability/master/index.html)

File: docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc
Comment (colleenmcginnis) [inline docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc:1]: The phrase "is a charged for service" sounds strange to me. What about something like this? Not sure if it should be "when you run monitors" or "when you create monitors":

```suggestion
When you run monitors on Elastic’s global managed testing infrastructure, you will be charged as outlined on the https://www.elastic.co/pricing[pricing page].
```

**Review by bmorelli25 — COMMENTED** [review]
I noticed that throughout the docs (including in this PR) we always refer to the feature as "Elastic’s global managed testing infrastructure". However, in the pricing guide, we use the term "managed global testing service"

```diff
-global managed testing infrastructure (docs)
-managed global testing service (pricing)
```

Not a huge deal, but it would be nice to update the pricing guide to use the same terminology that we're using in the docs.

File: docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc
Comment (paulb-elastic) [reply docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc:1]: Thanks @colleenmcginnis that's much better than what I had written as I wasn't sure what @drewpost wanted to include.

@drewpost are you happy to go with this, or something else?

**Comment by paulb-elastic** [comment]
> I noticed that throughout the docs (including in this PR) we always refer to the feature as "Elastic’s global managed testing infrastructure". However, in the pricing guide, we use the term "managed global testing service"
> 
> ```diff
> -global managed testing infrastructure (docs)
> -managed global testing service (pricing)
> ```
> 
> Not a huge deal, but it would be nice to update the pricing guide to use the same terminology that we're using in the docs.

Good spot, @drewpost can you help with this?

**Review by colleenmcginnis — APPROVED** [review]
I'm not sure if you're waiting on a review from @drewpost, but this looks good from my side.

**Comment by paulb-elastic** [comment]
> I'm not sure if you're waiting on a review from @drewpost, but this looks good from my side.

Thanks @colleenmcginnis yes, I just need Drew to confirm then we're good to merge.  @drewpost can you approve, or suggest alternative copy so we can merge this in.

File: docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc
Comment (drewpost) [reply docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc:1]: How about. "Executing synthetic tests on the Elastic global managed testing infrastructure incurs an additional charge. Tests are charged under one of two new billing dimensions depending on the type of test. Browser tests are charged for on a per test run basis and lightweight ping tests have an all you can eat model charged by region. Please refer to https://www.elastic.co/pricing for full details and current pricing" @paulb-elastic @colleenmcginnis

**Review by drewpost — CHANGES_REQUESTED** [review]

**Comment by paulb-elastic** [comment]
@drewpost (cc @colleenmcginnis) I've updated with your suggestion (tweaking the hyperlink text), and here's what it looks like:

See preview in https://observability-docs_3262.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-get-started-ui.html#uptime-set-up-prereq

image

See preview in https://observability-docs_3262.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-migrate-from-integration.html#synthetics-migrate-integration-location

image

See preview in https://observability-docs_3262.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-support-matrix.html

image

File: docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc
Comment (colleenmcginnis) [inline docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc:1]: A couple suggestions:

* Throughout the docs we tend to use "Elastic's global managed testing infrastructure" instead of "the Elastic global managed testing infrastructure".
* Throughout the docs we use "monitor types" instead of "type of test".
* The phrase "all you can eat model" might not translate well. I attempted a generic description that doesn't rely on familiarity with the idiom, but feel free to adjust.
* As a general rule, we avoid using "see" in favor of "refer to".

```suggestion
Executing synthetic tests on Elastic's global managed testing infrastructure incurs an additional charge. Tests are charged under one of two new billing dimensions depending on the monitor type. _Browser monitor_ usage is charged for on a per test run basis. _Lightweight monitor_ usage is charged a flat fee by region regardless of the number of runs. For more details, refer to https://www.elastic.co/pricing[full details and current pricing].
```

Comment (paulb-elastic) [reply docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc:1]: Thank you @colleenmcginnis, @drewpost, ok with this?

Comment (drewpost) [reply docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc:1]: @colleenmcginnis I'm fine with the standards type changes.

In terms of the lightweight tests, I think it's slightly confusing as if I see flat-fee on one but not the other, it makes me think there's some variable on the browser tests that isn't a flat rate. Could we use a slightly different term? FWIW, this unlimited consumption model is unique in the industry so we'd want to really emphasise it.

Comment (colleenmcginnis) [reply docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc:1]: >it makes me think there's some variable on the browser tests that isn't a flat rate

Isn't the number of test runs that variable? I guess maybe it depends on if there's a line item that says "Browser monitors" that is the sum of all runs or if each run is listed as a single line item? You know the pricing structure better than me so I trust your suggestions as long as we avoid idioms. 🙃  

Maybe as simple as "For _Lightweight monitor_ usage you're charged one fee per region regardless of the number of runs." ?

Comment (drewpost) [reply docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc:1]: image

This is what a customer would see in their cloud billing console. So for browser tests we charge a flat rate per test. So a customer running a test would be charged $0.0123 for up to 60 seconds of test execution. For lightweight tests, they simply are paying for capacity per region and can execute as many lightweight tests as they want within that capacity.

Comment (colleenmcginnis) [reply docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc:1]: Ok. How about something like this? 

```suggestion
Executing synthetic tests on Elastic's global managed testing infrastructure incurs an additional charge. Tests are charged under one of two new billing dimensions depending on the monitor type. For _browser monitor_ usage, there is a fee per test run. For _lightweight monitor_ usage, there is a fee per region in which you run any monitors regardless of the number of test runs. For more details, refer to https://www.elastic.co/pricing[full details and current pricing].
```

Comment (drewpost) [reply docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc:1]: Perfect @colleenmcginnis LGTM

**Review by colleenmcginnis — APPROVED** [review]

**Pull request merged**
